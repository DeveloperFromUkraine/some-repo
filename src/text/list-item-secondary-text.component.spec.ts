import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListItemSecondaryTextComponent } from "index";
import { ComponentTest } from "../../test/test-bed/component";

describe('ListItemSecondaryTextComponent', () => {
    let fixture: ComponentFixture<ListItemSecondaryTextComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [ListItemSecondaryTextComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListItemSecondaryTextComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});