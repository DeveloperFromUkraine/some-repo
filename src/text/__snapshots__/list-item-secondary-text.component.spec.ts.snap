// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItemSecondaryTextComponent should match snapshot 1`] = `
<ign-list-item-secondary-text>
  <h4
    _ngcontent-c0=""
    class="secondary-text"
    mat-line=""
  />
</ign-list-item-secondary-text>
`;
